packages/odyssey/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@okta/odyssey",
+  "private": true,
```

Comment:
This this temporary or are we no longer going to publish `@okta/odyssey`?

Reply:
the plan is to no longer publish this package yes, in fact we'll probably remove it from source control at some point in the near future and offer downstream teams options that better align with our revised architecture